wrappers/static/static.go

```diff
@@ -47,69 +56,94 @@ func (s *Wrapper) SetConfig(_ context.Context, opt ...wrapping.Option) (*wrappin
 
 	s.logger = opts.withLogger
 
+	// Load keys, identifiers from environments or options.
+	var currentKey, previousKey string
+
+	if os.Getenv(EnvBaoCurrentKeyValueName) != "" && !opts.Options.WithDisallowEnvVars {
+		currentKey = os.Getenv(EnvBaoCurrentKeyValueName)
+	} else {
+		currentKey = opts.withCurrentKey
+	}
+
+	if os.Getenv(EnvBaoCurrentKeyIdentifierName) != "" && !opts.Options.WithDisallowEnvVars {
+		s.currentKeyId = os.Getenv(EnvBaoCurrentKeyIdentifierName)
+	} else {
+		s.currentKeyId = opts.withCurrentKeyId
+	}
+
+	if os.Getenv(EnvBaoPreviousKeyValueName) != "" && !opts.Options.WithDisallowEnvVars {
+		previousKey = os.Getenv(EnvBaoPreviousKeyValueName)
+	} else {
+		previousKey = opts.withPreviousKey
+	}
+
+	if os.Getenv(EnvBaoPreviousKeyIdentifierName) != "" && !opts.Options.WithDisallowEnvVars {
+		s.previousKeyId = os.Getenv(EnvBaoPreviousKeyIdentifierName)
+	} else {
+		s.previousKeyId = opts.withPreviousKeyId
+	}
```

Comment:
Nit: I know this pattern reoccurs across the repo, couldn't it be much simpler?

```suggestion
	currentKey := opts.withCurrentKey
	previousKey := opts.withPreviousKey

	s.currentKeyId = opts.withCurrentKeyId
	s.previousKeyId = opts.withPreviousKeyId

	if !opts.Options.WithDisallowEnvVars {
		if env := os.Getenv(EnvBaoCurrentKeyValueName); env != "" {
			currentKey = env
		}
		if env := os.Getenv(EnvBaoPreviousKeyValueName); env != "" {
			previousKey = env
		}
		if env := os.Getenv(EnvBaoCurrentKeyIdentifierName); env != "" {
			s.currentKeyId = env
		}
		if env := os.Getenv(EnvBaoPreviousKeyIdentifierName); env != "" {
			s.previousKeyId = env
		}
	}
```